auto_dev/protocols/primitives.py

```diff
@@ -1,3 +1,7 @@
+"""Module containing custom primitives."""
+
+# ruff: noqa: D101, D102, D105, ARG003, PLW3201
+
```

Comment:
should add an explanation in comment, but the `D` ones are all about docstrings missing, `ARG003` is unused arguments and `PLW3201` is bad dunder method name, which is throws for `__get_pydantic_core_schema__` I implemented ([ruff rules](https://docs.astral.sh/ruff/rules/))